### PR TITLE
python3-CherryPy: update to 18.5.0.

### PR DESCRIPTION
**Pull request by bra1nwave**

### CODE REVIEW

File: srcpkgs/python-jaraco.functools/template [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
+revision=3
 archs=noarch
 wrksrc="jaraco.functools-${version}"
-build_style=python-module
-pycompile_module="jaraco.functools"
-hostmakedepends="python-setuptools python3-setuptools"
-depends="python-more-itertools python-backports.functools_lru_cache"
-short_desc="Additional functools in the spirit of Python2 stdlib's functools"
+build_style=python2-module
+pycompile_module="jaraco"
```
Comment (Chocimier) [inline srcpkgs/python-jaraco.functools/template]: This was ok, compiling jaraco.functools here and jaraco/init.py in jaraco.

File: srcpkgs/python-jaraco/template [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+pycompile_module="jaraco"
+short_desc="Namespace package declaration for jaraco (Python2)"
+maintainer="bra1nwave <bra1nwave@protonmail.com>"
+license="Public Domain"
+homepage="https://www.voidlinux.org/"
+
+do_install() {
+	for py_sitelib in $py2_sitelib $py3_sitelib; do
+		vmkdir ${py_sitelib}/jaraco
+		cat > ${DESTDIR}/${py_sitelib}/jaraco/__init__.py <<-EOF
```
Comment (Chocimier) [inline srcpkgs/python-jaraco/template]: Clever.
As we do not plan to add new py2 modules, this seems useful only for py3.

**Comment by bra1nwave** [comment]
I'm sorry it took me so long to answer. I was in Vacation.
Anyway, I've done all the proposed changes.

**Comment by Chocimier** [comment]
Please rebase so we'll see if they still build.

While here, please set pycompile_module to `jaraco-X` (or remove), `jaraco` is default now and not need to be set.

**Comment by bra1nwave** [comment]
Done. I removed all `pycompile_module` variables

**Pull request merged**